### ISSUE TITLE
Albedo satelite method

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/nowcasting_dataset#198]
Remove background satellite image

## Detailed Description
Remove background of satellite image by looking at the last 30 days, and minusing the mean/min/max or max value. This should be time of date depends. 

## Context
Good to remove general background and normalise the image

## Possible Implementation
- as described above
- create 30 day image, maybe mean, min and max of satellite images, and feed into model. Could also take 60, 90, ...360 image stats

**Comment by JackKelly** [comment]
Interesting idea!

Some discussion of similar ideas:

* https://github.com/openclimatefix/predict_pv_yield/issues/17
* https://github.com/openclimatefix/satflow/issues/6
* https://github.com/openclimatefix/satflow/issues/88

**Comment by jacobbieker** [comment]
Yeah, I think it'd be interesting! We could try using the cloud masks to subtract all non-cloudy pixels as well. The cloud masks for RSS only goes back till 2018, but could work as another way of doing it

**Comment by jacobbieker** [comment]
Another similar issue is https://github.com/openclimatefix/satflow/issues/100 as the Weather4cast data seems to be the optimum cloud product from EUMETSAT, as well as some of the other derived data products
